openupgrade_framework/odoo_patch/odoo/addons/base/models/ir_model.py

```diff
@@ -64,29 +62,6 @@ def _module_data_uninstall(self, modules_to_remove):
 IrModelData._module_data_uninstall = _module_data_uninstall
 
 
-@api.model
-def _process_end(self, modules):
-    """Don't warn about upgrade conventions from Odoo
-    ('fields should be explicitly removed by an upgrade script')
-    """
-    with mute_logger("odoo.addons.base.models.ir_model"):
-        return IrModelData._process_end._original_method(self, modules)
-
-
-_process_end._original_method = IrModelData._process_end
-IrModelData._process_end = _process_end
-
-
-def _module_data_uninstall(self):
```

Comment:
Are you sure this patch can be dropped? My comment on #4327 only hinted at removing the `_process_end` patch above.

Reply:
My bad, maybe it still need